x-pack/plugins/ml/public/application/datavisualizer/file_based/components/combined_fields/geo_point.tsx

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import _ from 'lodash';
```

Comment:
Only `debounce` is used, so could be
```suggestion
import debounce from 'lodash/debounce';
```